src/status_im/keycard/recovery.cljs

```diff
@@ -197,12 +198,27 @@
                   (navigation/navigate-to-cofx (if platform/android?
                                                  :notifications-settings :welcome) nil))))))
 
+(fx/defn return-to-keycard-login
+  [{:keys [db] :as cofx}]
+  (fx/merge cofx
+            {:db (-> db
+                     (assoc-in [:keycard :pin :enter-step] :login)
```

Comment:
you can probably do as well:
```
(update-in [:keycard :pin] assoc :enter-step :login
                                                   :status nil
                                                   :login [])
```
Which is a bit neater

Reply:
thanks for the review! fixed